README.md

```diff
@@ -1,10 +1,51 @@
 # the-crowd
-## Description :
 
```

Comment:
It should include the problem that we're trying to solve, and the solution that we get with the App description.